sematic/config/server_settings.py

```diff
@@ -0,0 +1,172 @@
+# Standard Library
+import enum
+import logging
+import os
+from typing import Dict, Optional
+
+# Sematic
+from sematic.config.config_dir import get_config_dir
+from sematic.config.settings import (
+    _as_bool,
+    _load_settings,
+    _normalize_enum,
+    _save_settings,
+)
+
+logger = logging.getLogger(__name__)
+
+_SERVER_SETTINGS_FILE = "server-settings.yaml"
+
+
+class ServerSettingsVar(enum.Enum):
```

Comment:
Should the S3 bucket be in this list? I think the cleanest way to do it would be to have the server maintain the configuration for where to write, and have the client query the server when it needs to know where to write. This may dovetail nicely with some work currently in progress for @neutralino1

Reply:
FWIW, the server does need to know about this bucket so it can read logs from it

Reply:
How about duplicating the setting in both enums until the feature in which the storage is only accessed through the server is implemented, a point in which it can be removed from the user settings?

Reply:
sounds good, particularly since that feature is about to land

Reply:
Duplicated this setting. Made the server settings the default access for it, with a fallback to user settings. Introduced deprecation TODOs.